src/UI/Factory.php

```diff
@@ -261,15 +261,15 @@ public function image();
 	 *     This component is used to wrap an existing ILIAS UI element into a UI component. This is useful if a container
 	 *     of the UI components needs to contain content that is not yet implement in the centralized UI components.
 	 *   composition: >
-	 *     The generic component contains html or any other content as string.
+	 *     The legacy component contains html or any other content as string.
 	 *
 	 * rules:
 	 *   wording:
 	 *      1: This component MUST only be used to ensure backwards compatibility with existing UI elements in ILIAS.
```

Comment:
I'm perfectly fine with SHOULD in this context since we can't predict all use cases like nesting